src/App.js

```diff
@@ -0,0 +1,62 @@
+import "macro-css";
```

Comment:
what is macro-css?

Reply:
a plugin that allows to write such styles as display: flex simply d-flex in the class